src/provider/element-templates/Validator.js

```diff
@@ -210,3 +246,45 @@ export function filteredSchemaErrors(schemaErrors) {
     return false;
   });
 }
+
+
+// helpers ///////////////////////
+
+const moddle = new BpmnModdle();
```

Comment:
I see.. https://github.com/bpmn-io/bpmn-js-properties-panel/pull/648/commits/2fe461897b40b89667cd3c88e35f75b3e0835ad2#r841676679.

Reply:
My proposal here is: Consider re-using the existing `moddle` instance, and its introspection capabilities: https://github.com/bpmn-io/moddle/blob/master/lib/moddle.js#L200.

Reply:
> Consider re-using the existing `moddle` instance

I did not consider this at first, as it was a standalone utility (no dependency injection). I will look into passing it to the constructor

Reply:
...aaaand forgot to move the dependency back to dev :see_no_evil:

Reply:
> ...aaaand forgot to move the dependency back to dev see_no_evil

done with [b378bd2](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/648/commits/b378bd2fb78a9a4a0ffba95e5d27f87542de3ec6)